packages/react/src/components/Toast/ToastProvider.tsx

```diff
@@ -96,16 +96,19 @@ export const ToastProvider = ({ children }: PropsWithChildren) => {
     const invert = last - first;
 
     // PLAY
-    container.current!.animate(
-      [
-        { transform: `translateY(${invert}px)` },
-        { transform: "translateY(0)" },
-      ],
-      {
-        duration: getSlideInDuration(),
-        easing: "ease",
-      },
-    );
+    // We need to check if the animate function exists because it's not available in some testing environments ( like jest-dom )
+    if (typeof container.current!.animate === "function") {
```

Comment:
There is a chance that `container.current` is null:
```suggestion
    if (container.current && typeof container.current.animate === "function") {
```